Makefile

```diff
@@ -0,0 +1,13 @@
+flake:
+	flake8 --max-line-length=120
+
+tests-unit:
```

Comment:
Is there a way to make it verbose and to see the name of tests running?

Reply:
Yes, in fact, it is possible to use `tox -e unit -- --debug`, do you think it would be relevant to add another make command like `tests-unit-verbose` (with the same suffix for others),

Reply:
Oh and there is also `--verbose`!

Reply:
I suggest to let the verbose option by default, as it doesn't seem to flood that much.

Reply:
Okay cool, thank you! Would you kindly add the `tox -e unit -- --debug` command to the readme, so we have the command for running tests with tox with the full output?

Reply:
I mean to the contributing doc :D